comment/managers/comments.py

```diff
@@ -16,8 +16,13 @@ def all_exclude_flagged(self):
 
         return super().get_queryset().exclude(flag__state__exact=2)
 
+    @staticmethod
+    def _filter_parents(qs, parent=None):
+        return qs.filter(parent=parent).order_by(validate_order())
```

Comment:
return list from `validate_order()`
```suggestion
        return qs.filter(parent=parent).order_by(*validate_order())
```